backend/src/supervisors/supervisor.py

```diff
@@ -25,30 +25,29 @@ def solve_task(task, scratchpad, attempt=0) -> Tuple[str | None, str]:
     if agent is None:
         return (None, no_agent_response)
 
-    answer = agent.invoke(task["summary"])
-    if is_valid_answer(answer, task):
-        return (agent.name, answer)
+    answer = agent.invoke(task)
+    # if is_valid_answer(answer, task):
```

Comment:
hmm, I wonder if the validator agent needs to be more light touch for now, and then something we can experiment on in the future e.g. do we get an answer, if so great lets say success ?